Formula/daemontools.rb

```diff
@@ -22,6 +22,40 @@ def install
     end
   end
 
+  def caveats; <<~EOS
+    You must create /service directory as root before starting svscan.
```

Comment:
What is `svscan` used for? Is Daemontools useful without it?

Reply:
The `svscan` is daemontools' core program.

As its name suggests, daemontools is a set of tools for daemon programs, and svscan is the command to supervise daemon programs in the "daemotools world". First of all, we can not use daemontools' set of tools to not use svscan.

Reply:
Each of our few daemontools users on homebrew has installed plist on their own. For example, it is as follows.

https://gist.github.com/bobpp/824501

However, daemontools in MacPorts automatically start svscan.
https://github.com/macports/macports-ports/blob/master/sysutils/daemontools/Portfile